<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>362</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Leelawadee UI</family>
    <pointsize>10</pointsize>
    <stylestrategy>PreferDefault</stylestrategy>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>MrSquigy's Belt Builder</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblNote">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>265</y>
      <width>241</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee UI</family>
      <pointsize>10</pointsize>
      <stylestrategy>PreferDefault</stylestrategy>
     </font>
    </property>
    <property name="text">
     <string>Don't forget you can use up to 3 boss cards!</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="indent">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btnReset">
    <property name="geometry">
     <rect>
      <x>254</x>
      <y>265</y>
      <width>83</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee UI</family>
      <pointsize>10</pointsize>
      <stylestrategy>PreferDefault</stylestrategy>
     </font>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QWidget" name="slotsParent">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>11</y>
      <width>325</width>
      <height>246</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QFormLayout" name="laySlots">
       <item row="0" column="0">
        <widget class="QLabel" name="lblSlot1">
         <property name="font">
          <font>
           <family>Leelawadee UI</family>
           <pointsize>10</pointsize>
           <stylestrategy>PreferDefault</stylestrategy>
          </font>
         </property>
         <property name="text">
          <string>Slot 1</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="cbxSlot1">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Leelawadee UI</family>
           <pointsize>10</pointsize>
           <stylestrategy>PreferDefault</stylestrategy>
          </font>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblSlot2">
         <property name="font">
          <font>
           <family>Leelawadee UI</family>
           <pointsize>10</pointsize>
           <stylestrategy>PreferDefault</stylestrategy>
          </font>
         </property>
         <property name="text">
          <string>Slot 2</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="cbxSlot2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Leelawadee UI</family>
           <pointsize>10</pointsize>
           <stylestrategy>PreferDefault</stylestrategy>
          </font>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lblSlot3">
         <property name="font">
          <font>
           <family>Leelawadee UI</family>
           <pointsize>10</pointsize>
           <stylestrategy>PreferDefault</stylestrategy>
          </font>
         </property>
         <property name="text">
          <string>Slot 3</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="cbxSlot3">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Leelawadee UI</family>
           <pointsize>10</pointsize>
           <stylestrategy>PreferDefault</stylestrategy>
          </font>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lblSlot4">
         <property name="font">
          <font>
           <family>Leelawadee UI</family>
           <pointsize>10</pointsize>
           <stylestrategy>PreferDefault</stylestrategy>
          </font>
         </property>
         <property name="text">
          <string>Slot 4</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="cbxSlot4">
         <property name="minimumSize">
          <size>
           <width>160</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Leelawadee UI</family>
           <pointsize>10</pointsize>
           <stylestrategy>PreferDefault</stylestrategy>
          </font>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lblSlot5">
         <property name="font">
          <font>
           <family>Leelawadee UI</family>
           <pointsize>10</pointsize>
           <stylestrategy>PreferDefault</stylestrategy>
          </font>
         </property>
         <property name="text">
          <string>Slot 5</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="cbxSlot5">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Leelawadee UI</family>
           <pointsize>10</pointsize>
           <stylestrategy>PreferDefault</stylestrategy>
          </font>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="lblSlot6">
         <property name="font">
          <font>
           <family>Leelawadee UI</family>
           <pointsize>10</pointsize>
           <stylestrategy>PreferDefault</stylestrategy>
          </font>
         </property>
         <property name="text">
          <string>Slot 6</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="cbxSlot6">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Leelawadee UI</family>
           <pointsize>10</pointsize>
           <stylestrategy>PreferDefault</stylestrategy>
          </font>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="lblSlot7">
         <property name="font">
          <font>
           <family>Leelawadee UI</family>
           <pointsize>10</pointsize>
           <stylestrategy>PreferDefault</stylestrategy>
          </font>
         </property>
         <property name="text">
          <string>Slot 7</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QComboBox" name="cbxSlot7">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Leelawadee UI</family>
           <pointsize>10</pointsize>
           <stylestrategy>PreferDefault</stylestrategy>
          </font>
         </property>
         <item>
          <property name="text">
           <string>Boss Cards Only</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="lblSlot8">
         <property name="font">
          <font>
           <family>Leelawadee UI</family>
           <pointsize>10</pointsize>
           <stylestrategy>PreferDefault</stylestrategy>
          </font>
         </property>
         <property name="text">
          <string>Slot 8</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QComboBox" name="cbxSlot8">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Leelawadee UI</family>
           <pointsize>10</pointsize>
           <stylestrategy>PreferDefault</stylestrategy>
          </font>
         </property>
         <item>
          <property name="text">
           <string>Boss Cards Only</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QFormLayout" name="layStages">
       <item row="0" column="0">
        <widget class="QLabel" name="lblStage1">
         <property name="font">
          <font>
           <family>Leelawadee UI</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
           <stylestrategy>PreferDefault</stylestrategy>
          </font>
         </property>
         <property name="text">
          <string>Stage</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="cbxStage1">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Leelawadee UI</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
           <stylestrategy>PreferDefault</stylestrategy>
          </font>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblStage2">
         <property name="font">
          <font>
           <family>Leelawadee UI</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
           <stylestrategy>PreferDefault</stylestrategy>
          </font>
         </property>
         <property name="text">
          <string>Stage</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="cbxStage2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Leelawadee UI</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
           <stylestrategy>PreferDefault</stylestrategy>
          </font>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lblStage3">
         <property name="font">
          <font>
           <family>Leelawadee UI</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
           <stylestrategy>PreferDefault</stylestrategy>
          </font>
         </property>
         <property name="text">
          <string>Stage</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="cbxStage3">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Leelawadee UI</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
           <stylestrategy>PreferDefault</stylestrategy>
          </font>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lblStage4">
         <property name="font">
          <font>
           <family>Leelawadee UI</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
           <stylestrategy>PreferDefault</stylestrategy>
          </font>
         </property>
         <property name="text">
          <string>Stage</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="cbxStage4">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Leelawadee UI</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
           <stylestrategy>PreferDefault</stylestrategy>
          </font>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lblStage5">
         <property name="font">
          <font>
           <family>Leelawadee UI</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
           <stylestrategy>PreferDefault</stylestrategy>
          </font>
         </property>
         <property name="text">
          <string>Stage</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="cbxStage5">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Leelawadee UI</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
           <stylestrategy>PreferDefault</stylestrategy>
          </font>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="lblStage6">
         <property name="font">
          <font>
           <family>Leelawadee UI</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
           <stylestrategy>PreferDefault</stylestrategy>
          </font>
         </property>
         <property name="text">
          <string>Stage</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="cbxStage6">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Leelawadee UI</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
           <stylestrategy>PreferDefault</stylestrategy>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="statsParent">
    <property name="geometry">
     <rect>
      <x>356</x>
      <y>13</y>
      <width>316</width>
      <height>341</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="layStats">
     <item row="3" column="1">
      <widget class="QLabel" name="lblVit">
       <property name="font">
        <font>
         <family>Leelawadee UI</family>
         <pointsize>10</pointsize>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
       <property name="text">
        <string>Vitality</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="lblDex">
       <property name="font">
        <font>
         <family>Leelawadee UI</family>
         <pointsize>10</pointsize>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
       <property name="text">
        <string>Dexterity</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="lblCrit">
       <property name="font">
        <font>
         <family>Leelawadee UI</family>
         <pointsize>10</pointsize>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
       <property name="text">
        <string>Crit Rate</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lblStr">
       <property name="font">
        <font>
         <family>Leelawadee UI</family>
         <pointsize>10</pointsize>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
       <property name="text">
        <string>Strength</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="lblEva">
       <property name="font">
        <font>
         <family>Leelawadee UI</family>
         <pointsize>10</pointsize>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
       <property name="text">
        <string>Evasion</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QLabel" name="lblMpReg">
       <property name="font">
        <font>
         <family>Leelawadee UI</family>
         <pointsize>10</pointsize>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
       <property name="text">
        <string>MP Regen</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblStatBenefits">
       <property name="font">
        <font>
         <family>Leelawadee UI</family>
         <pointsize>10</pointsize>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
       <property name="text">
        <string>Stat Benefits</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QLabel" name="lblHpReg">
       <property name="font">
        <font>
         <family>Leelawadee UI</family>
         <pointsize>10</pointsize>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
       <property name="text">
        <string>HP Regen</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLabel" name="lblMdef">
       <property name="font">
        <font>
         <family>Leelawadee UI</family>
         <pointsize>10</pointsize>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
       <property name="text">
        <string>M. Defense</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QLabel" name="lblCastSpd">
       <property name="font">
        <font>
         <family>Leelawadee UI</family>
         <pointsize>10</pointsize>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
       <property name="text">
        <string>Cast Spd</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="lblPdef">
       <property name="font">
        <font>
         <family>Leelawadee UI</family>
         <pointsize>10</pointsize>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
       <property name="text">
        <string>P. Defense</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="lblPatk">
       <property name="font">
        <font>
         <family>Leelawadee UI</family>
         <pointsize>10</pointsize>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
       <property name="text">
        <string>P. Attack</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="lblInt_">
       <property name="font">
        <font>
         <family>Leelawadee UI</family>
         <pointsize>10</pointsize>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
       <property name="text">
        <string>Int</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="lblLuck">
       <property name="font">
        <font>
         <family>Leelawadee UI</family>
         <pointsize>10</pointsize>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
       <property name="text">
        <string>Luck</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="lblMatk">
       <property name="font">
        <font>
         <family>Leelawadee UI</family>
         <pointsize>10</pointsize>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
       <property name="text">
        <string>M. Attack</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="lblWis">
       <property name="font">
        <font>
         <family>Leelawadee UI</family>
         <pointsize>10</pointsize>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
       <property name="text">
        <string>Wisdom</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="lblAgi">
       <property name="font">
        <font>
         <family>Leelawadee UI</family>
         <pointsize>10</pointsize>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
       <property name="text">
        <string>Agility</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="lblAtkSpd">
       <property name="font">
        <font>
         <family>Leelawadee UI</family>
         <pointsize>10</pointsize>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
       <property name="text">
        <string>Atk Spd</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="lblPacc">
       <property name="font">
        <font>
         <family>Leelawadee UI</family>
         <pointsize>10</pointsize>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
       <property name="text">
        <string>P. Accuracy</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="lblMacc">
       <property name="font">
        <font>
         <family>Leelawadee UI</family>
         <pointsize>10</pointsize>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
       <property name="text">
        <string>M. Accuracy</string>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QLabel" name="lblBlockPer">
       <property name="font">
        <font>
         <family>Leelawadee UI</family>
         <pointsize>10</pointsize>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
       <property name="text">
        <string>Block Per</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLabel" name="lblMres">
       <property name="font">
        <font>
         <family>Leelawadee UI</family>
         <pointsize>10</pointsize>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
       <property name="text">
        <string>M. Res</string>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="lblBlockDef">
       <property name="font">
        <font>
         <family>Leelawadee UI</family>
         <pointsize>10</pointsize>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
       <property name="text">
        <string>Block Def</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="lblBeltEnhance">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>305</y>
      <width>116</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Belt Enhancement</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbxBeltEnhance">
    <property name="geometry">
     <rect>
      <x>255</x>
      <y>300</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>0-19</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>20</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>21</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>22</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>23</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>24</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>25</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>cbxSlot1</tabstop>
  <tabstop>cbxStage1</tabstop>
  <tabstop>cbxSlot2</tabstop>
  <tabstop>cbxStage2</tabstop>
  <tabstop>cbxSlot3</tabstop>
  <tabstop>cbxStage3</tabstop>
  <tabstop>cbxSlot4</tabstop>
  <tabstop>cbxStage4</tabstop>
  <tabstop>cbxSlot5</tabstop>
  <tabstop>cbxStage5</tabstop>
  <tabstop>cbxSlot6</tabstop>
  <tabstop>cbxStage6</tabstop>
  <tabstop>cbxSlot7</tabstop>
  <tabstop>cbxSlot8</tabstop>
  <tabstop>btnReset</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
